core/io/resource_importer.cpp

```diff
@@ -456,6 +457,18 @@ void ResourceFormatImporter::get_dependencies(const String &p_path, List<String>
 	ResourceLoader::get_dependencies(pat.path, p_dependencies, p_add_types);
 }
 
+void ResourceFormatImporter::get_build_dependencies(const String &p_path, HashSet<String> *r_dependencies) {
+	if (!exists(p_path)) {
+		return;
+	}
+
+	List<Ref<ResourceImporter>> valid_importers;
+	get_importers_for_file(p_path, &valid_importers);
+	for (Ref<ResourceImporter> importer : valid_importers) {
+		return importer->get_build_dependencies(p_path, r_dependencies);
```

Comment:
MSVC gives a warning for unreachable code here, but the fact this is returning at all seems like a relic of a previous iteration. I assume this is instead meant to be:
```suggestion
		importer->get_build_dependencies(p_path, r_dependencies);
```